corehq/apps/app_manager/xform.py

```diff
@@ -503,6 +503,9 @@ def add_case_updates(self, updates, make_relative=False):
         update_mapping = {}
         attachments = {}
         for key, value in updates.items():
+            from corehq.apps.app_manager.models import ConditionalCaseUpdate
```

Comment:
The plan is that eventually `value` will always be a `ConditionalCaseUpdate`, right?